### PR TITLE
Add new node keys in rococo chain

### PR DESCRIPTION
**Pull request by sudipghimire533**
Add new sets of keys in rococo chain

### CODE REVIEW

File: res/rococo/rococo-parachain-state [hunk 1]
```diff
@@ -1 +1 @@
-0x000000000000000000000000000000000000000000000000000000000000000000cf53fa329f52a92f921debd7e725942371da41a31047009a4feb9cc4d02d96b403170a2e7597b7b7e3d84c05391d139a62b157e78786d8c082f29dcf4c11131400
```
Comment (penumbra23) [inline res/rococo/rococo-parachain-state:1]: Is this change needed or just a local build change?

**Review by penumbra23 — APPROVED** [review]

File: res/rococo/rococo-parachain-state (hunk 1 above)
Comment (sudipghimire533) [reply res/rococo/rococo-parachain-state:1]: It is needed for this release. This is actual state registered in rococo. We might want to delete whole `res` dir after #49

**Review by iTranscend — APPROVED** [review]
